servers/rendering/renderer_rd/storage_rd/texture_storage.cpp

```diff
@@ -1921,6 +1993,362 @@ Ref<Image> TextureStorage::_validate_texture_format(const Ref<Image> &p_image, T
 	return image;
 }
 
+void TextureStorage::_texture_format_from_rd(RD::DataFormat p_rd_format, TextureFromRDFormat &r_format) {
+	switch (p_rd_format) {
+		case RD::DATA_FORMAT_R8_UNORM: {
```

Comment:
Do we want to support more formats like the BGR ones here?

Reply:
We should support all the formats that we currently support in the RS

Reply:
Yeah that is what this code is based on, I basically took `_validate_texture_format` and reversed it.
However RD supports many more formats that in theory could be used here.

But I suggest we keep it as is, and if anyone ever runs into a real live use case for using other formats, we'll add the required entries.